CONTRIBUTING.md

```diff
@@ -6,9 +6,11 @@ and follow the [guidelines](https://jazzband.co/about/guidelines).
 
 ## Project Contribution Guidelines
 
-Here are a few additional or emphasized guidelines to follow when contributing to pip-tools:
+Here are a few additional or emphasized guidelines to follow when contributing to `pip-tools`:
 
-- Install pip-tools in development mode and its test dependencies with `pip install -e .[testing]`.
+- To set up your project's environment, have a look at [Python Packaging User Guide](https://packaging.python.org)
+  and [Real Python Tutorials](https://realpython.com)
```

Comment:
Could you link to specific articles rather than website homepages?

Reply:
I couldn't find any good article about `tox` so I provided a link to tox's documentation instead.